### PR TITLE
BaSP-TG-130: Solved general bugs

### PR DESCRIPTION
**Pull request by agusting19**
In this PR we solved bugs in general.
-

### CODE REVIEW

**Review by Johsua30 — APPROVED** [review]
Great!

File: src/Components/LandingPage/index.jsx [hunk 1]
```diff
@@ -69,8 +69,9 @@ function Home() {
             <div>
               <h3 className={styles.subtitle}>Multiple Roles</h3>
               <p>
-                Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestiae eligendi
-                consequatur inventore asperiores illo atque at.
+                we are commited to flexible work and to building tools that fit into any workflow.
```
Comment (Javier-Cervera) [inline src/Components/LandingPage/index.jsx:72]: We

**Review by Javier-Cervera — APPROVED** [review]
Looks good

**Review by NicolasFontana — APPROVED** [review]
Looks great!

**Pull request merged**